EssentialsDiscord/src/main/java/net/essentialsx/discord/JDADiscordService.java

```diff
@@ -192,7 +192,7 @@ public void startup() throws LoginException, InterruptedException {
         }
 
         // Load emotes into cache, JDA will handle updates from here on out.
-        guild.retrieveEmotes().queue();
+        guild.retrieveEmojis().queue();
```

Comment:
Refer to my previous point. JDA usually does cache emojis when it gets the Guild into the cache.

Reply:
ditto